### ISSUE TITLE
Input Group Prepend/Append not working

### ISSUE THREAD

**Issue by tylerpolzin** [bootstrap-ruby/bootstrap_form#405]
I'm not sure what's changed in the past week, but I'm in the middle of building an app and both the "prepend" and "append" features are suddenly broken for me.  

I did a test fix on my webpage, and replacing the individual class text with "input-group-addon" instead of "input-group-prepend" worked.  Can there be an update to allow "addon" in the rails helper?  I suppose the helper would also need to indicate if the addon is happening before or after the input.  I'd rather not fuss with fixing all of my addons manually!

**Comment by lcreid** [comment]
Thanks for reporting this, Tyler. I'm sorry this broke for you. Can you please confirm the version of `bootstrap_form` you're using, and the version of the Bootstrap gem you're using (or the version of Bootstrap you're including, if you're not using the Bootstrap gem)? While you're at it, please give us the Rails and Ruby version you're using. 

We're in the middle of some major revisions to `bootstrap_form`, so it would help us to know the versions you're working with.

Thanks in advance.

**Comment by tylerpolzin** [comment]
Sure, 

Using bootstrap_form 4.0.0.dev from https://github.com/bootstrap-ruby/rails-bootstrap-forms.git (at master@4c8c178)
Using bootstrap-sass 3.3.7
ruby 2.3.4p301 (2017-03-30 revision 58214) [x86_64-linux]
Rails 5.1.4

I'll keep an eye on this page.  Thanks for the quick update!

**Comment by tylerpolzin** [comment]
I just had a bit of an "oh no..." moment when I realized that I'm not even on Bootstrap 4.  I hadn't paid attention in about a year, and assumed it was updating automatically for me on every bundle update, without realizing 'bootstrap-sass' is no longer managed.  I just made a quick attempt to install the 'bootstrap' gem instead of 'bootstrap-sass' and it completely borked my entire site's CSS.  Am I SOL with Bootstrap Form by not being on 4?
